sonar/modules/deposits/jsonschemas/deposits/deposit-v1.0.0_src.json

```diff
@@ -839,12 +838,199 @@
             "minLength": 1,
             "pattern": "^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$",
             "form": {
-              "placeholder": "Example: 1111-2222-3333-4444"
+              "templateOptions": {
+                "placeholder": "Example: 1111-2222-3333-4444"
+              }
             }
           }
         }
       }
     },
+    "projects": {
+      "title": "Research projects",
+      "type": "array",
```

Comment:
minItems seems missing.

Reply:
Is that mandatory if array can be empty?